.github/workflows/run-tests.yaml

```diff
@@ -31,3 +31,18 @@ jobs:
         with:
           name: Report
           path: ${{ env.REPORT_PATH }}
+
+  test-documentation:
+    name: Test documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y mandoc scdoc ninja-build
+
+      - name: Test Documentation
+        run: |
+          ./doc/configure
+          ninja
```

Comment:
Since you're not actually running tests, I would prefer to call this `build-documentation`

Reply:
`test-build-documentation`? To make it clear that you're testing whether the documentation can be successfully built. 🚢 otherwise, thank you!